smarts/core/smarts.py

```diff
@@ -1053,9 +1053,15 @@ def _try_emit_envision_state(self, provider_state, obs, scores):
             for bubble in self._bubble_manager.bubbles
         ]
 
+        scenario_name = {
+            agent_id: (self.scenario._root).split("/")[-1]
```

Comment:
A path split library could be used instead of using `.split("/")[-1]`.

Reply:
Appropriate changes implemented. Thanks 👍